src/components/user/user-profile.js

```diff
@@ -1,12 +1,43 @@
 import React from 'react';
-import { Paper } from '@material-ui/core';
+import { Paper, AppBar, Tabs, Tab } from '@material-ui/core';
+import { withStyles } from '@material-ui/core/styles';
 
 import { user } from '@openchemistry/girder-ui';
 
-export default function BasicInfo() {
+
+const StyledTabs = withStyles({
+  indicator: {
+    display: 'flex',
+    justifyContent: 'center',
+    backgroundColor: 'transparent',
+    '& > div': {
+      width: '100%',
+      backgroundColor: '#ffffff',
+    },
+  },
```

Comment:
Great to us use `withStyles(...)`

Reply:
Totally agree with keeping this very simple, our target audience will likely have no idea what Girder is, and we should use a simple set of options where reasonable.